app/src/main/java/org/mozilla/focus/perf/Performance.kt

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.perf
+
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.os.BatteryManager
+import org.mozilla.focus.state.AppAction
+import org.mozilla.focus.state.AppStore
+import android.provider.Settings as AndroidSettings
+
+object Performance {
+
+    private const val EXTRA_IS_PERFORMANCE_TEST = "performancetest"
+
+    fun processIntentIfPerformanceTest(intent: Intent, context: Context, appStore: AppStore) {
```

Comment:
I think that would be better in case it gets refactored / loss. Also, any change in one might be needed in the other. I think its just better for overall code health to have the duplicated code in one place